eclair-core/src/main/resources/reference.conf

```diff
@@ -99,9 +99,10 @@ eclair {
       claim-main = 12    // target for the claim main transaction (tx that spends main channel output back to wallet)
     }
 
-    // maximum local vs remote feerate mismatch; 1.0 means 100%
-    // actual check is abs((local feerate - remote fee rate) / (local fee rate + remote fee rate)/2) > fee rate mismatch
-    max-feerate-mismatch = 1.56 // will allow remote fee rates up to 8x bigger or smaller than our local fee rate
+    feerate-mismatch {
+      ratio-low = 0.5 // will allow remote fee rates as low as half our local feerate
+      ratio-high = 8.0 // will allow remote fee rates as high as 8 times our local feerate
+    }
```

Comment:
```suggestion
    feerate-tolerance {
      ratio-low = 0.5 // will allow remote fee rates as low as half our local feerate
      ratio-high = 8.0 // will allow remote fee rates as high as 8 times our local feerate
    }
```

Reply:
Fixed in https://github.com/ACINQ/eclair/pull/1473/commits/9037da2978266fb0de40bf13c272d0024dfedcea